docs/learning-path.mdx

```diff
@@ -24,6 +24,7 @@ For information on how variants tie in to the levels, see the [variant summary](
 ### Level Summary
 
 The following is a quick summary of the levels.
+You can also see a [cheat sheet of many of the main conventions](summary.mdx).
```

Comment:
this won't appear on a newline, is that intended? if not, then put it on the same line